cvat/apps/engine/mixins.py

```diff
@@ -217,32 +236,55 @@ def append_tus_chunk(self, request, file_id):
                                       extra_headers={'Upload-Offset': tus_file.offset,
                                                      'Upload-Filename': tus_file.filename})
 
-    def validate_filename(self, filename):
+    def validate_uploaded_file_name(self, filename: str) -> bool:
+        """
+        Checks the file name to be valid.
```

Comment:
It would be useful to explain what it means for a file name to be valid.

Reply:
Probably it's an implementation detail/